package.json

```diff
@@ -12,7 +12,7 @@
     "private::lint.renderer": "cd dash-renderer && npm run lint",
     "private::test.setup-nested": "cd \\@plotly/dash-generator-test-component-nested && npm ci && npm run build && pip install -e .",
     "private::test.setup-standard": "cd \\@plotly/dash-generator-test-component-standard && npm ci && npm run build && pip install -e .",
-    "private::test.unit-dash": "PYTHONPATH=~/dash/tests/assets pytest tests/unit",
```

Comment:
was breaking for me since i didn't clone dash into `~/`

Reply:
heh nor did I - looks like this was originally hard-coded in our circleci setup, then moved out of there when we created a dash-level `package.json` in #1151 but not generalized. TBH locally I've always run the desired tests directly via `pytest` but this is certainly better 🎉 

Actually it would be cleaner to do something inside `unit/test_app_runners.py` rather than rely on `PYTHONPATH` on the command line. It'd be nice if you could just run `pytest` and have it work. Apparently that was never the case for this test... I wonder if we could just `chdir` into `tests/assets`? `sys.path` won't work because the app gets started in a new process.

Anyway @dannysepler not something you need to worry about unless you're motivated to try. I'm just thinking out loud.

Reply:
totally agreed! big fan of just running `pytest` -- it's a lot more intuitive for new peeps like me. unfortunately i _still_ can't get the tests working lol (attached an issue about it), so i'm scared to do anything beyond the super surface-level